src/main/java/gov/usgs/owi/nldi/controllers/LinkedDataController.java

```diff
@@ -157,15 +157,16 @@ public void getCharacteristicData(HttpServletRequest request, HttpServletRespons
 			String comid = getComid(featureSource, featureID);
 
 			if (null == comid) {
-				response.sendError(HttpStatus.NOT_FOUND.value(), "Feature not found");
+				response.setStatus(HttpStatus.NOT_FOUND.value());
+			} else {
+				Map<String, Object> parameterMap = new HashMap<> ();
+				parameterMap.put(Parameters.CHARACTERISTIC_TYPE, characteristicType.toLowerCase());
+				parameterMap.put(Parameters.COMID, NumberUtils.parseNumber(comid, Integer.class));
+				parameterMap.put(Parameters.CHARACTERISTIC_ID, characteristicIds);
+				addContentHeader(response);
+				streamResults(transformer, BaseDao.CHARACTERISTIC_DATA, parameterMap);
 			}
```

Comment:
This fixes the issue here and also addresses an error:  attempting to stream content should only happen if the comid is not null.